boms/legacy-ee/pom.xml

```diff
@@ -1362,6 +1362,12 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
+                <groupId>org.jboss.spec.javax.batch</groupId>
+                <artifactId>jboss-batch-api_1.0_spec</artifactId>
+                <version>${version.org.jboss.spec.javax.batch.jboss-batch-api_1.0_spec}</version>
+            </dependency>
```

Comment:
This looks like it got added after a rebase. It should be removed.

Reply:
Yes sir! I'll remove it.